index.html

```diff
@@ -4,10 +4,9 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title></title>
-	<link rel="stylesheet" href="https://binary-studio-academy.github.io/stage-2/assets/stylesheets/base.css">
+    <title>Deadpool qoutes</title>
```

Comment:
Title was changed and there was no info about this

Reply:
Sorry my bad